.eslintrc.js

```diff
@@ -1,12 +0,0 @@
-module.exports = {
```

Comment:
I would rather have it in a `.eslintrc.js` @vvo

Reply:
@CBaptiste no problemo, for the record I now use package.json for every configuration when doable so that the number of different files at the root of the project is low. But understand if that's the best practice of your team so I will move it.